apps/server/src/modules/user/uc/dto/user.dto.ts

```diff
@@ -25,7 +25,7 @@ export class UserDto {
 
 	roleIds: string[] = [];
 
-	schoolId: string;
+	schoolId?: string;
```

Comment:
the school is not an optional field for the user entity. it would also break the implementation of the user service.

Reply:
done